README.md

```diff
@@ -59,5 +59,10 @@ The use the "create pack" button from the app to convert such a directory to a `
 You can then install the resulting pack to your device with the "install pack" button.
 
 ---
+## Build
+Build in GO and React, desktop app powered by [Wails](https://github.com/wailsapp/)
```

Comment:
There is a typo from me here, it was intended to be "built with" - mainly to pay my respects to wails, so i think it should stay at the end of the general section.